working/templates/package-project/GettingStarted.md

```diff
@@ -0,0 +1,154 @@
+# Getting Started with Skyline DataMiner DevOps
+
+Welcome to the Skyline DataMiner DevOps environment!  
+This quick-start guide will help you get up and running.  
+For more details and comprehensive instructions, please visit [DataMiner Docs](https://docs.dataminer.services/).
+
+<!--#if (CreateDataMinerPackage)-->
+## Creating a DataMiner Application Package
+
+This project is configured to create a `.dmapp` file every time you build the project.  
+Simply compile or build the project, and you will find the generated `.dmapp` in the standard output folder, typically the `bin` folder of your project.
+
+When publishing, this project will become its own item on the online catalog.
+
+## The DataMiner Package Project
+
+This project is designed to create multi-artifact packages in a straightforward manner.
+
+### Other DataMiner Projects in the Same Solution
+
+Every **Skyline.DataMiner.SDK** project, except other DataMiner Package Projects, will by default be included within the `.dmapp` created by this project.  
+You can customize this behavior using the **PackageContent/ProjectReferences.xml** file. This allows you to add filters to include or exclude projects as needed.
+
+### Adding Content from the Catalog
+
+You can reference and include additional content from the catalog using the **PackageContent/CatalogReferences.xml** file provided in this project.
+
+### Importing from DataMiner
+
+You can import specific items directly from a DataMiner agent:  
+
+1. Connect to an agent via **Extensions > DIS > DMA > Connect**.
+2. Once connected, you can import specific DataMiner artifacts.
+3. Navigate to folders such as **PackageContent/Dashboards** or **PackageContent/LowCodeApps**, right-click, select **Add**, and choose **Import DataMiner Dashboard/LowCodeApp** or the equivalent.
+
+## Execute Additional Code on Installation
+
+Open the **$SCRIPTNAME$.cs** file to write custom installation code. Common actions include creating elements, services, or views.
+
+**Quick Tip:** Type `clGetDms` in the `.cs` file and press **Tab** twice to insert a snippet that gives you access to the **IDms** classes, making DataMiner manipulation easier.
+
+## Does Your Installation Code Need Configuration Files?
+
+You can add configuration files (e.g., `.json`, `.xml`) to the **SetupContent** folder, which can be accessed during installation.
```

Comment:
We don't usually use a comma after "e.g." (house style), so while it's not wrong to use it, I'd recommend removing it here for the sake of consistency.

Reply:
Applied